spec/MultiDomainInventoryProxy+config.json

```diff
@@ -179,26 +179,26 @@
             },
             "response-profile-configuration": {
               "value": {
-                "value-reference": "/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-0-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address"
+                "value-reference": "/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address"
               }
             }
           }
         },
         {
-          "uuid": "mdip-1-0-0-response-p-007",
```

Comment:
For /v1/inform-about-release-history-in-generic-representation , response-profile needs to be included for the all the releases. Currently its for only the 1st release.